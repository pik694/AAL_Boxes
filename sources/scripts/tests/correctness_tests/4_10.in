4
11 43 3
40 44 91
74 55 28
91 58 9

