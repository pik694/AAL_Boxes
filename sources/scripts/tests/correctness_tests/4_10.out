0 3 2 1 
323041->321622
